src/shared/prompt.ts

```diff
@@ -1,27 +0,0 @@
-/*
```

Comment:
I renamed it to `promptUtils` to match the pattern of the rest of the files in that location but for some reason diff is showing it as deleted instead of renamed.